src/components/Common/Export.tsx

```diff
@@ -79,7 +79,7 @@ export const ExportMenu = ({
         disabled={isExporting || disabled}
         title={isExporting ? "Exporting..." : label}
         icon={<CareIcon icon="l-export" />}
-        className="tooltip border-primary-500 bg-white text-primary-500 hover:bg-primary-100 enabled:border"
+        className="tooltip border-primary-700 bg-white text-primary-700 hover:bg-primary-100 enabled:border"
       >
         {exportItems.map((item) => (
           <DropdownItem
```

Comment:
Let's switch to dropdown from shadcn too since you've touched this file

https://ui.shadcn.com/docs/components/dropdown-menu